_literate/EX-breastcancer/tutorial.jl

```diff
@@ -183,13 +183,12 @@ for m in models(matching(X, y))
         y_pred = MLJ.predict(clf_machine, rows=test);
 
         #Plotting the ROC-AUC curve for each model being iterated
-        fprs, tprs, thresholds = roc(y_pred, y[test])
-        plot(fprs, tprs,label=model_name);
-
+        fprs, tprs, thresholds = roc_curve(y_pred, y[test])
+        plot!(p, fprs, tprs,label=model_name);
         #Obtaining different evaluation metrics
         ce_loss = mean(cross_entropy(y_pred,y[test]))
         acc = accuracy(mode.(y_pred), y[test])
-        f1_score = f1score(mode.(y_pred), y[test])
+        f1_score = MLJ.f1score(mode.(y_pred), y[test])
```

Comment:
Why do we need `MLJ.` qualification here? I can't imagine any other package here is exporting the name. Is this to clarify for the student which package is providing the method? An alternative is a comment that MLJ re-exports all the measures in [`StatisticalMeasures.jl`](https://juliaai.github.io/StatisticalMeasures.jl/dev/) 

FYI
```julia
julia> using MLJ

julia> f1score
FScore(
  beta = 1.0, 
  levels = nothing, 
  rev = nothing, 
  checks = true)

(jl_IHpzTB) pkg> status
Status `/private/var/folders/4n/gvbmlhdc8xj973001s6vdyw00000gq/T/jl_IHpzTB/Project.toml`
  [add582a8] MLJ v0.20.3

```

Reply:
I remember getting an error that it couldn't find `f1score` and that was my fix. Now tried running again without it and it worked. My Julia installation has to be haunted.